### PR TITLE
CNV_3045_2 Add NIC to VM Files specific to 4.2

### PR DESCRIPTION
**Pull request by bgaydosrh**
Checking Build....

### CODE REVIEW

**Comment by bgaydosrh** [comment]
The content in this PR has been code reviewed and QE'd.

It does need a peer review then merge and pick to enterprise-4.2 ONLY.

Please label Peer Review Needed and Enterprise-4.2.

Thanks,
Bob

File: modules/cnv-add-nic-to-vm.adoc [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+= Adding a network interface to a {object}
+
+.Procedure
+
+. From the *{object-gui}* tab, select the {object} you created.
+. Select the *Network Interfaces* tab.
+. Click *Add Network Interface*.
+. In the *Add Network Interface* window, specify the *Name*, *Model*, *Network*, *Type*,
+and *MAC Address* of the network interface.
+. Click *Add Network Interface* to add the network interface without restarting the virtual
```
Comment (yossisegev) [inline modules/cnv-add-nic-to-vm.adoc]: I'm having issues in opening the web console, so I can't see or try it for myself, but this section seems weird: The user already clicked "Add Network Interface" 2 sections ago, and in the previous section he was still in the "Add Network Interface" window, so it's seems odd he should click "Add Network Interface" again.
Please re-check.

File: modules/cnv-add-nic-to-vm.adoc [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+:object-gui: Virtual Machine Templates
+endif::[]
+
+[id="cnv-vm-add-nic_{context}"]
+
+= Adding a network interface to a {object}
+
+.Procedure
+
+. From the *{object-gui}* tab, select the {object} you created.
```
Comment (ousleyp) [inline modules/cnv-add-nic-to-vm.adoc]: I don't think "you created" is necessary here.

File: modules/cnv-add-nic-to-vm.adoc [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+. Select the *Network Interfaces* tab.
+. Click *Add Network Interface*.
+. In the *Add Network Interface* window, specify the *Name*, *Model*, *Network*, *Type*,
+and *MAC Address* of the network interface.
+. Click *Add Network Interface* to add the network interface without restarting the virtual
+machine. Note that for the virtual machine to be accessible, it must be restarted.
```
Comment (ousleyp) [inline modules/cnv-add-nic-to-vm.adoc]: Maybe: "You must restart the virtual machine before you can access it."

File: modules/cnv-add-nic-to-vm.adoc [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+. Use the drop-down lists and check boxes to edit the network
```
Comment (ousleyp) [inline modules/cnv-add-nic-to-vm.adoc]: Similar nit to the other PR. I'd prefer "Edit the ____ by ____"

File: modules/cnv-add-nic-to-vm.adoc [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+interface configuration.
+. Click *Save Changes*.
+. Click *OK*.
+
+The new network interface is listed at the top of the list of network interfaces
+until the user restarts it. The new network interface has a ’Pending VM restart’
+*Link State* until you restart the virtual machine. Hovering over the
+*Link State* will display more detailed information.
```
Comment (ousleyp) [inline modules/cnv-add-nic-to-vm.adoc]: This section is a bit wordy and could be rephrased into the active voice. Examples:

"You can find the new network interface at the top of the (is there a UI element you could reference here?) list."

"The new network interface has a `Pending VM restart` **Link State** until you restart the virtual machine. Hover over the **Link State** to display more detailed information."

(`Pending VM restart` should be formatted with backticks, not single quotes.)

**Review by ousleyp — COMMENTED** [review]
A few minor comments. Otherwise, this is looking good.

File: modules/cnv-add-nic-to-vm.adoc (hunk 1 above)
Comment (jelkosz) [reply modules/cnv-add-nic-to-vm.adoc]: on 4.3 it is:
On the nic list there is an "Add Network Interface" button to open the add nic dialog. After you fill in all the details, you need to submit the dialog; that submit button is called "Add"

on 4.2 it is:
On the nic list there is an "Create Nic" button. If you click it, not a dialog but an editable row will show up. To submit it, you need to press the check-mark (:heavy_check_mark:) button on the right hand side of the row.

File: modules/cnv-add-nic-to-vm.adoc (hunk 2 above)
Comment (bgaydosrh) [reply modules/cnv-add-nic-to-vm.adoc]: Done.

File: modules/cnv-add-nic-to-vm.adoc (hunk 3 above)
Comment (bgaydosrh) [reply modules/cnv-add-nic-to-vm.adoc]: Added a new step: " Restart the virtual machine to enable access."

File: modules/cnv-add-nic-to-vm.adoc (hunk 4 above)
Comment (bgaydosrh) [reply modules/cnv-add-nic-to-vm.adoc]: Good one. Done.

File: modules/cnv-add-nic-to-vm.adoc (hunk 5 above)
Comment (bgaydosrh) [reply modules/cnv-add-nic-to-vm.adoc]: Cool beans. Done.

**Review by bgaydosrh — COMMENTED** [review]
Review complete

File: modules/cnv-add-nic-to-vm.adoc (hunk 1 above)
Comment (bgaydosrh) [reply modules/cnv-add-nic-to-vm.adoc]: For 4.2 changed these steps to:

. From the *{object-gui}* tab, select the {object}.
. Select the *Network Interfaces* tab.
. Click *Create Network Interface*.
. In the *Create Network Interface* list row, specify the
*Name*, *Model*, *Network*, *Type*,
and *MAC Address* of the network interface.
. Click the green check-mark to the right of the row to add the network interface.
. Restart the virtual machine to enable access.
. Edit the drop-down lists and check boxes to configure the network
interface.
. Click *Save Changes*.
. Click *OK*.
